### ISSUE TITLE
BUG: 3500 lines of messages about Emerald and Trade (Log)

### ISSUE THREAD

**Issue by Krimatoria** [Lightman314/LightmansCurrency#200]
### Minecraft Version

1.19.2

### Forge Version

Forge 43.3.5

### Lightman's Currency Version

lightmanscurrency-1.19.2-2.2.0.3c.jar

### Environment Type

Minecraft Client (Bug/crash happened in a single-player world or while the game was booting)

### Describe the bug

It's just an endless replay of the registration of some Emerald. I have 2500 such lines in the log.
![image](https://github.com/Lightman314/LightmansCurrency/assets/106809386/b1c2e45c-cc7d-4639-8cbc-809eda04e0c1)

![image](https://github.com/Lightman314/LightmansCurrency/assets/106809386/b5a25313-6a3a-4612-96f7-dbbf46e9da61)

### To Reproduce

[Log 3.txt](https://github.com/Lightman314/LightmansCurrency/files/14313435/Log.3.txt)

### Screenshots/Logs/Crash Reports

There are still a lot of such messages, the log is littered with them too.

![image](https://github.com/Lightman314/LightmansCurrency/assets/106809386/14ffeb43-9e96-4c5f-b902-2dded0f5470b)

[16Feb2024 19:00:38.761] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/cypress/green as it's serializer returned null
[16Feb2024 19:00:38.761] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/skyris/yellow as it's serializer returned null
[16Feb2024 19:00:38.762] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/bookshelf/biomesoplenty/mahogany as it's serializer returned null
[16Feb2024 19:00:38.762] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe engineeredcompatibility:sawmill/ironwood_bookshelf as it's serializer returned null
[16Feb2024 19:00:38.762] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/fir/purple as it's serializer returned null
[16Feb2024 19:00:38.762] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/willow/magenta as it's serializer returned null
[16Feb2024 19:00:38.762] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/white_mangrove/orange as it's serializer returned null
[16Feb2024 19:00:38.762] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/white_mangrove/lime as it's serializer returned null
[16Feb2024 19:00:38.765] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf2/biomesoplenty/dead as it's serializer returned null
[16Feb2024 19:00:38.769] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/cherry/green as it's serializer returned null
[16Feb2024 19:00:38.772] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:auction_stand/byg/palm as it's serializer returned null
[16Feb2024 19:00:38.772] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/palm/pink as it's serializer returned null
[16Feb2024 19:00:38.772] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/fir/cyan as it's serializer returned null
[16Feb2024 19:00:38.775] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/baobab/light_blue as it's serializer returned null
[16Feb2024 19:00:38.775] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/aspen/lime as it's serializer returned null
[16Feb2024 19:00:38.775] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/dead/purple as it's serializer returned null
[16Feb2024 19:00:38.776] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/maple/light_blue as it's serializer returned null
[16Feb2024 19:00:38.776] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/bookshelf/byg/ether as it's serializer returned null
[16Feb2024 19:00:38.776] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/palm/purple as it's serializer returned null
[16Feb2024 19:00:38.777] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/cherry/brown as it's serializer returned null
[16Feb2024 19:00:38.777] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/witch_hazel/blue as it's serializer returned null
[16Feb2024 19:00:38.777] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/fungal_imparius/red as it's serializer returned null
[16Feb2024 19:00:38.778] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/nightshade/red as it's serializer returned null
[16Feb2024 19:00:38.778] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/willow/blue as it's serializer returned null
[16Feb2024 19:00:38.778] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/holly/green as it's serializer returned null
[16Feb2024 19:00:38.779] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf2/biomesoplenty/umbran as it's serializer returned null
[16Feb2024 19:00:38.779] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/maple/cyan as it's serializer returned null
[16Feb2024 19:00:38.780] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf/byg/skyris as it's serializer returned null
[16Feb2024 19:00:38.780] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/jacaranda/purple as it's serializer returned null
[16Feb2024 19:00:38.780] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/cypress/red as it's serializer returned null
[16Feb2024 19:00:38.781] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/nightshade/pink as it's serializer returned null
[16Feb2024 19:00:38.781] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/embur/yellow as it's serializer returned null
[16Feb2024 19:00:38.785] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/sythian/light_blue as it's serializer returned null
[16Feb2024 19:00:38.785] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/bulbis/red as it's serializer returned null
[16Feb2024 19:00:38.785] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/willow/orange as it's serializer returned null
[16Feb2024 19:00:38.786] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/redwood/cyan as it's serializer returned null
[16Feb2024 19:00:38.787] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/imparius/cyan as it's serializer returned null
[16Feb2024 19:00:38.789] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/ebony/magenta as it's serializer returned null
[16Feb2024 19:00:38.789] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/holly/black as it's serializer returned null
[16Feb2024 19:00:38.789] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/jacaranda/brown as it's serializer returned null
[16Feb2024 19:00:38.790] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/rainbow_eucalyptus/light_gray as it's serializer returned null
[16Feb2024 19:00:38.790] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/willow/lime as it's serializer returned null
[16Feb2024 19:00:38.790] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/witch_hazel/lime as it's serializer returned null
[16Feb2024 19:00:38.790] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/sythian/white as it's serializer returned null
[16Feb2024 19:00:38.790] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/palm/cyan as it's serializer returned null
[16Feb2024 19:00:38.790] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/mahogany/cyan as it's serializer returned null
[16Feb2024 19:00:38.791] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/cherry/orange as it's serializer returned null
[16Feb2024 19:00:38.791] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/mahogany/light_gray as it's serializer returned null
[16Feb2024 19:00:38.791] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/pine/purple as it's serializer returned null
[16Feb2024 19:00:38.793] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe supplementaries:inspirations/flag_dye as it's serializer returned null
[16Feb2024 19:00:38.794] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/mahogany/green as it's serializer returned null
[16Feb2024 19:00:38.794] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/rainbow_eucalyptus/black as it's serializer returned null
[16Feb2024 19:00:38.794] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/ether/red as it's serializer returned null
[16Feb2024 19:00:38.794] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf2/byg/cherry as it's serializer returned null
[16Feb2024 19:00:38.794] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/blue_enchanted/green as it's serializer returned null
[16Feb2024 19:00:38.794] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/cypress/orange as it's serializer returned null
[16Feb2024 19:00:38.795] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/magic/purple as it's serializer returned null
[16Feb2024 19:00:38.795] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/blue_enchanted/white as it's serializer returned null
[16Feb2024 19:00:38.797] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf2/byg/cypress as it's serializer returned null
[16Feb2024 19:00:38.798] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/aspen/magenta as it's serializer returned null
[16Feb2024 19:00:38.798] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/magic/pink as it's serializer returned null
[16Feb2024 19:00:38.798] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/cypress/brown as it's serializer returned null
[16Feb2024 19:00:38.798] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf2/byg/ether as it's serializer returned null
[16Feb2024 19:00:38.799] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/imbued_green_enchanted/green as it's serializer returned null
[16Feb2024 19:00:38.799] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe engineeredcompatibility:crusher/wilden_horn as it's serializer returned null
[16Feb2024 19:00:38.799] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:auction_stand/byg/maple as it's serializer returned null
[16Feb2024 19:00:38.799] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/lament/yellow as it's serializer returned null
[16Feb2024 19:00:38.799] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/rainbow_eucalyptus/white as it's serializer returned null
[16Feb2024 19:00:38.799] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/mahogany/gray as it's serializer returned null
[16Feb2024 19:00:38.802] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/bookshelf/byg/jacaranda as it's serializer returned null
[16Feb2024 19:00:38.803] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/rainbow_eucalyptus/green as it's serializer returned null
[16Feb2024 19:00:38.803] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/hellbark/purple as it's serializer returned null
[16Feb2024 19:00:38.803] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/redwood/cyan as it's serializer returned null
[16Feb2024 19:00:38.803] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/imbued_green_enchanted/cyan as it's serializer returned null
[16Feb2024 19:00:38.803] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/bookshelf/byg/blue_enchanted as it's serializer returned null
[16Feb2024 19:00:38.804] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/ether/yellow as it's serializer returned null
[16Feb2024 19:00:38.805] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/rainbow_eucalyptus/blue as it's serializer returned null
[16Feb2024 19:00:38.805] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/sythian/yellow as it's serializer returned null
[16Feb2024 19:00:38.806] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:auction_stand/byg/willow as it's serializer returned null
[16Feb2024 19:00:38.806] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/mahogany/black as it's serializer returned null
[16Feb2024 19:00:38.806] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/maple/magenta as it's serializer returned null
[16Feb2024 19:00:38.807] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/zelkova/light_gray as it's serializer returned null
[16Feb2024 19:00:38.807] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/skyris/gray as it's serializer returned null
[16Feb2024 19:00:38.808] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/imbued_green_enchanted/black as it's serializer returned null
[16Feb2024 19:00:38.809] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/bookshelf/byg/fir as it's serializer returned null
[16Feb2024 19:00:38.809] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/white_mangrove/black as it's serializer returned null
[16Feb2024 19:00:38.810] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf/biomesoplenty/mahogany as it's serializer returned null
[16Feb2024 19:00:38.811] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/maple/orange as it's serializer returned null
[16Feb2024 19:00:38.812] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/embur/lime as it's serializer returned null
[16Feb2024 19:00:38.812] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/shelf/byg/cherry as it's serializer returned null
[16Feb2024 19:00:38.813] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe engineeredcompatibility:fermenter/redcurrant as it's serializer returned null
[16Feb2024 19:00:38.813] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/withering_oak/light_gray as it's serializer returned null
[16Feb2024 19:00:38.817] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe engineeredcompatibility:fermenter/whitecurrant as it's serializer returned null
[16Feb2024 19:00:38.821] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/blue_enchanted/black as it's serializer returned null
[16Feb2024 19:00:38.822] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/rainbow_eucalyptus/lime as it's serializer returned null
[16Feb2024 19:00:38.822] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/skyris/light_gray as it's serializer returned null
[16Feb2024 19:00:38.822] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/imbued_nightshade/pink as it's serializer returned null
[16Feb2024 19:00:38.822] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/mahogany/brown as it's serializer returned null
[16Feb2024 19:00:38.822] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/biomesoplenty/palm/purple as it's serializer returned null
[16Feb2024 19:00:38.823] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/zelkova/magenta as it's serializer returned null
[16Feb2024 19:00:38.823] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/green_enchanted/magenta as it's serializer returned null
[16Feb2024 19:00:38.823] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Skipping loading recipe lightmanscurrency:traders/card_display/byg/withering_oak/gray as it's serializer returned null

**Comment by Lightman314** [comment]
I cannot stop the ones about the card displayes, as those are posted by forge about the conditional recipes (as the recipes for my wood compats are conditional, requiring the mod providing said wood be loaded) and cannot be blocked on my end. That is an issue you'd need to take up with forge themselves sadly.

The logs about the villager trades can and will be fixed in the next update though.

**Comment by Krimatoria** [comment]
> I cannot stop the ones about the card displays, as those are posted by forge about the conditional recipes failing their condition (as the recipes for my wood compats are conditional, requiring the mod providing said wood be loaded) and cannot be blocked on my end. That is an issue you'd need to take up with forge themselves sadly.
> 
> The logs about the villager trades can and will be fixed in the next update though.

I didn't quite understand what you meant. I don't need to deal with any question) I just saw some errors and posted them on GitHub) I'm not a mod developer, I'm just a player.

**Comment by Lightman314** [comment]
TLDR:
* Ignore the logs about the card displays. They're going to be there if another mod isn't installed, but they cause no actual harm yet are out of my capabilities to actually remove and prevent the log-spam.
* The logs about the villager trades will be removed in the future to prevent unwanted log spam.

**Comment by Lightman314** [comment]
Fixed in v2.2.1.0.
